toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -277,6 +277,17 @@ func (c *Chroot) AddFiles(filesToCopy ...FileToCopy) (err error) {
 	return
 }
 
+// ExtractFile copies file 'srcPath' in the chroot to the host at 'destPath'
+func (c *Chroot) ExtractFile(srcPath string, destPath string) (err error) {
+	srcPathFull := filepath.Join(c.rootDir, srcPath)
+	err = file.Copy(srcPathFull, destPath)
```

Comment:
Do we want to optionally clean up this file?

Reply:
Sure, sounds like a good idea

Reply:
Just my suggestion regarding how we can implenent it: to keep the functions simple, I'd suggest having two separate `CopyOut` and `MoveOut` functions instead of adding an extra argument, which the user will now always have to worry about.